contentcuration/contentcuration/management/commands/mark_incomplete.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Exists
+from django.db.models import OuterRef
+from django.db.models import Q
+from le_utils.constants import content_kinds
+from le_utils.constants import exercises
+
+from contentcuration.models import AssessmentItem
+from contentcuration.models import ContentNode
+from contentcuration.models import File
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        exercise_check_query = AssessmentItem.objects.filter(contentnode=OuterRef('id')) \
+            .exclude(type=exercises.PERSEUS_QUESTION)\
```

Comment:
What are perseus questions actually?

Reply:
[Perseus](https://github.com/Khan/perseus) is the exercise format used by Khan Academy (and under the hood, it's what we use to render our own exercises). These questions have a lot more capabilities than other questions because they take more advantage of the widgets available